hieradata/org/lsst/role/dev_hyp.yaml

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
Is there a difference between `dev_hyp` and `hypervisor`?

Reply:
dev_hyp is for the server that is acting as core2

Reply:
We have a `core2.dev.lsst.org`? And how does this vary from the `hypervisor` role?